README.md

```diff
@@ -247,14 +247,29 @@ TraversedPartialPath: Could not traverse through HexaryTrieNode(sub_segments=((0
     except TraversedPartialPath as exc:
       last_exception = exc
 
-# We can now drive the exploration of the extension node further, by back-forming sub-segments:
->>> sub_segments = [
-      last_exception.nibbles_traversed + segment
-      for segment in last_exception.node.sub_segments
-]
+# We can now continue exploring the children of the extension node, by using an attribute on the exception:
+>>> sub_segments = last_exception.simulated_node.sub_segments
+((0x6, 0x5, 0x7, 0x9),)
+# Note that this sub-segment now carries you the rest of the way to the child
```

Comment:
I notice the file has mixed narrative perspective anyway but since most of it seems to be following the "we" form I'm going to point out this last grammar nit before I try to understand what's actually going on here haha.

>Note that this sub-segment now carries ~~you~~ us the rest of the way to the child of the node that ~~you~~ we only partially traversed into.